lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -162,6 +162,29 @@ namespace Js
         return Is_NoTaggedIntCheck(object) && TryGetInt32Value(GetValue(object), &i) ? i : 0;
     }
 
+    double JavascriptNumber::DirectPowDoubleInt(double x, int32 y)
```

Comment:
This is referenced as `IR::HelperDirectMath_PowDoubleInt` in JIT.

---
In reply to: [57507059](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57507059) [](ancestors = 57507059)